<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FoodListForm</class>
 <widget class="QWidget" name="FoodListForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,1">
   <item>
    <widget class="QLineEdit" name="foodFilterlineEdit">
     <property name="placeholderText">
      <string>Начните вводить название</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="sortButtonsLayout" stretch="4,1,1,1,1">
     <item>
      <widget class="QPushButton" name="sortByNameButton">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Название</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sortByPbutton">
       <property name="text">
        <string>Белки</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sortByFbutton">
       <property name="text">
        <string>Жиры</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sortByCHbutton">
       <property name="text">
        <string>Углеводы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sortByCalButton">
       <property name="text">
        <string>Ккал / 100 г</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>644</width>
        <height>504</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
